Microsoft.Toolkit.Win32/Microsoft.Toolkit.Sample.Wpf.App/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Resources;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Windows;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Windows Community Toolkit Controls (Sample Xaml Island Wrapper Controls for .NET WPF)")]
+[assembly: AssemblyDescription("Sample Code for Windows Community Toolkit Controls")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyProduct("Windows Community Toolkit")]
+[assembly: AssemblyCopyright("Copyright © Microsoft 2018")]
```

Comment:
>Copyright © Microsoft 2018 [](start = 30, length = 26)

This should read "Copyright .NET Foundation and Contributors. All rights reserved."